project/src/components/app/app.tsx

```diff
@@ -1,5 +1,11 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import MainPage from '../main-page/main-page';
+
+type AppScreenProps = {
```

Comment:
```suggestion
type AppProps = {
``` Но это чисто мое имхо))